src/MangaJet/app/src/main/java/com/mangajet/mangajet/mangareader/MangaReaderViewModel.kt

```diff
@@ -77,7 +115,7 @@ class MangaReaderViewModel : ViewModel() {
     }
 
     // Function which will upload pages
-    fun uploadPages() {
+    private fun uploadPages() {
         for (job in jobs)
             if (job != null && job.isActive)
```

Comment:
I think job?.cancel() will be better. Because
If job active it will be canceled
If job done it wont be canceled
If job canceled cancel wont do anything
If job null no cancel will be called